.github/SUPPORT.md

```diff
@@ -7,9 +7,10 @@ Node.js](https://github.com/nodejs/Release#release-schedule).
 When looking for support, please first search for your question in these venues:
 
 * [Node.js Website](https://nodejs.org/en/), especially the
-  [API docs](https://nodejs.org/api/)
-* [Node.js Help](https://github.com/nodejs/help)
+  [API docs](https://nodejs.org/api/) or the [Learn material](https://nodejs.org/en/learn)
+* [Node.js's GitHub Help repository](https://github.com/nodejs/help)
 * [Open or closed issues in the Node.js GitHub organization](https://github.com/issues?utf8=%E2%9C%93&q=sort%3Aupdated-desc+org%3Anodejs+is%3Aissue)
+* [Ask for support on Node.js's Discord server](https://nodejs.org/discord)
```

Comment:
what about slack and irc? if any of these 3 are mentioned anywhere, i think all 3 must be mentioned.

Reply:
These communities are currently unofficial and have been active for many years. I kindly ask that we refrain from discussing their potential to become official in my pull requests and other forums. I appreciate your passion on the topic, but this is the fifth time we've touched on this since I joined Node, and I think it might be helpful to focus on other matters for now. Thank you for your understanding!

Reply:
I'm very surprised that Slack (specifically the OpenJS F Slack instance) is considered "unofficial".  We refer to it in several places, e.g. https://github.com/nodejs/node/blob/109c817db63197870c72d8f8e25e151684fa1277/doc/contributing/pull-requests.md?plain=1#L47-L56

Reply:
@richardlau the OpenJS Slack is official. What @ljharb is referring here is his unofficial Node Slackers Slack community.

Reply:
The links you mentioned above are for OpenJS's Slack and not "Node Slackers"